### DISCUSSION TITLE
N1 Openwrt无法安装1.2.5-1_aarch64_cortex-a53

### DISCUSSION THREAD

**Discussion by murazi** [messense/aliyundrive-webdav#328] [Q&A] [answered]
### 问题描述

~# opkg install aliyundrive-webdav_1.2.5-1_aarch64_cortex-a53.ipk

Unknown package 'aliyundrive-webdav'.
Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.

安装提示不兼容，这个应该是N1的版本没错吧？A53

### 重现步骤

重复几遍，安装错误一样，强制安装也不行

### 版本

1.2.5-1_aarch64_cortex-a53

### 运行平台

斐讯N1-Openwrt

### 日志

Unknown package 'aliyundrive-webdav'.
Collected errors:
 * pkg_hash_fetch_best_installation_candidate: Packages for aliyundrive-webdav found, but incompatible with the architectures configured
 * opkg_install_cmd: Cannot install package aliyundrive-webdav.

**Comment by messense** [comment, marked as answer]
> Tips: 不清楚 CPU 架构类型可通过运行 opkg print-architecture 命令查询。

**Comment by lawzhougc** [comment]
遇到一样的情况  最后安装了aarch64_generic版本成功
